index.js

```diff
@@ -1,5 +1,44 @@
 'use strict'
 
+function code(state, startLine, endLine/*, silent*/) {
+  var nextLine, last, token, match
+  if (state.sCount[startLine] - state.blkIndent < 4) { return false }
+  last = nextLine = startLine + 1
+  while (nextLine < endLine) {
+    if (state.sCount[nextLine] - state.blkIndent >= 4) {
+      nextLine++
+      last = nextLine
+      continue
+    }
+    break
+  }
+  state.line = last
+  token         = state.push('indented_quote', 'blockquote', 0)
+  token.content = state.getLines(startLine, last, 4 + state.blkIndent, true)
+  match = token.content.match(/^ +/)
+  if (match) {
+    token.content = token.content.replace(/^ +/, '')
+    if (match[0].length >= 4) {
+      if (!token.attrs) token.attrs = {}
+      token.attrs.indent = 1 + Math.floor(match[0].length / 4)
```

Comment:
As I recall there is a `token.meta` (see [here](https://markdown-it.github.io/markdown-it/#Token.prototype.meta)) that should be used for storing arbitrary data instead of token.attrs; this way it will not be rendered.

On second thought, since you're going to have to use _open and _close tags, why not just set `` token.attrs.class = `indent-${indentLevel}` ``? Then I don't think you'll even need a custom render function, as the default should do fine. I'm not sure about this though.

Reply:
Fixed